app/src/main/java/com/kks/nimblesurveyjetpackcompose/util/CustomKeyGenerator.kt

```diff
@@ -0,0 +1,16 @@
+package com.kks.nimblesurveyjetpackcompose.util
+
+import android.content.Context
+import android.content.pm.ApplicationInfo
+import android.content.pm.PackageManager
+
+class CustomKeyGenerator(applicationContext: Context) : CustomKeyProvider {
```

Comment:
Shall we rename it to `CustomKeyProviderImpl` for parity with other implementation classes? 😄